_posts/elements/2015-04-09-buttons.md

```diff
@@ -17,77 +17,77 @@ description:  A collection of buttons available for using within the edX platfor
     <button type="button" class="btn btn-primary btn-base" disabled>Primary Base</button>
 </section>
 
+<h3 class="hd-6 example-set-hd">Secondary</h3>
 <section class="example-set">
-    <h3 class="hd-6">Secondary</h3>
     <button type="button" class="btn btn-secondary btn-base">Secondary Base</button>
     <button type="button" class="btn btn-secondary btn-large">Secondary Large</button>
     <button type="button" class="btn btn-secondary btn-small">Secondary Small</button>
     <button type="button" class="btn btn-secondary btn-base" disabled>Secondary Base</button>
 </section>
 
+<h3 class="hd-6 example-set-hd">Secondary (Alt)</h3>
 <section class="example-set">
-    <h3 class="hd-6">Secondary (Alt)</h3>
     <button type="button" class="btn btn-secondary-alt btn-base">Secondary Base</button>
     <button type="button" class="btn btn-secondary-alt btn-large">Secondary Large</button>
     <button type="button" class="btn btn-secondary-alt btn-small">Secondary Small</button>
     <button type="button" class="btn btn-secondary-alt btn-base" disabled>Secondary Base</button>
 </section>
 
+<h3 class="hd-6 example-set-hd">Links</h3>
 <section class="example-set">
-    <h3 class="hd-6">Links</h3>
     <button type="button" class="btn btn-link btn-base">Link Base</button>
     <button type="button" class="btn btn-link btn-large">Link Large</button>
     <button type="button" class="btn btn-link btn-small">Link Small</button>
     <button type="button" class="btn btn-link btn-base" disabled>Link Base</button>
 </section>
 
+<h3 class="hd-6 example-set-hd">Elevated</h3>
 <section class="example-set">
-    <h3 class="hd-6">Elevated</h3>
     <button type="button" class="btn btn-elevated btn-base">Elevated Base</button>
     <button type="button" class="btn btn-elevated btn-large">Elevated Large</button>
     <button type="button" class="btn btn-elevated btn-small">Elevated Small</button>
     <button type="button" class="btn btn-elevated btn-base" disabled>Elevated Base</button>
 </section>
 
+<h3 class="hd-6 example-set-hd">Combo</h3>
 <section class="example-set">
-    <h3 class="hd-6">Combo</h3>
     <button type="button" class="btn btn-primary btn-base btn-combo">Combo Base</button>
     <button type="button" class="btn btn-primary btn-base btn-combo">Combo Base</button>
     <button type="button" class="btn btn-primary btn-base btn-combo">Combo Base</button>
 </section>
 
-<!--<section class="example-set">
-    <h3 class="hd-6">With Icons</h3>
+<!--<h3 class="hd-6 example-set-hd">With Icons</h3>
+<section class="example-set">
     <button type="button" class="btn btn-default btn-base has-icon">Default Base with Icon</button>
     <button type="button" class="btn btn-default btn-base has-icon icon-reversed">Default Base with Icon Reversed</button>
 </section>-->
 
+<h3 class="hd-6 example-set-hd">Overlay</h3>
 <section class="example-set">
-    <h3 class="hd-6">Overlay</h3>
```

Comment:
Valid, accessible markup requires a heading to be the first child of a `section`. If we want these headings to be outside, we should convert the `sections` to `div`'s.